Sources/ApolloSQLite/SQLiteDatabase.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+#if !COCOAPODS
+import Apollo
+#endif
+
+public struct DatabaseRow {
+  let cacheKey: CacheKey
+  let storedInfo: String
+}
+
+public protocol SQLiteDatabase {
```

Comment:
Would like to see the functions on this protocol very well documented. If someone wants to inject a DIFFERENT SQLLite dependency in the future, they should be able to easily understand what their adapter needs to do in each of these functions.

Reply:
oh for sure - i was thinking i'd do that when i hit the final version but it's probably just as well to do it now given my goldfish brain

Reply:
As long as we do it at some point. If you want to do it later, go ahead and merge this. Up to you!